aosp_kyleprods.mk

```diff
@@ -6,5 +6,5 @@ PRODUCT_NAME := aosp_kyleprods
 
 # Set build fingerprint / ID / Product Name etc.
 PRODUCT_BUILD_PROP_OVERRIDES += \
-    PRODUCT_NAME=aosp_kyleprods \
-    TARGET_DEVICE=kyleprods \
\ No newline at end of file
+    PRODUCT_NAME=kyleprods \
```

Comment:
If you are changing PRODUCT_NAME to kyleprods, .mk should be changed too vice versa otherwise you will have error in lunch. So leave it too